bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be replaced by an `OH-INF/addons/addons.xml` file, see #13289

Reply:
resolved